packages/kotti-ui/source/kotti-table/components/TableBodyExpandRow.ts

```diff
@@ -18,9 +18,7 @@ export const TableBodyExpandRow: any = {
 				h(
 					'td',
 					{
-						domProps: {
-							colSpan,
```

Comment:
https://github.com/vuejs/vue/blob/13f4e7dc03e2caed900ac70ff8b8fe58dda45663/packages/server-renderer/src/util.ts#L6

Reply:
fixed and verified in a local build